### ISSUE TITLE
clarification on joystick range signs

### ISSUE THREAD

**Issue by ChavoDeLa** [sastejugaad/Deskbot-MecanumWheel#1]
for the following equations to be true:

Speed = Ly;
Strafe = Lx;
Turn = Rx;
LeftFrontWheel = Speed + Strafe - Turn;
RightFrontWheel = Speed - Strafe - Turn;
LeftBackWheel = Speed - Strafe + Turn;
RightBackWheel = Speed + Strafe - Turn;

must Lx, Ly, and Rx joystick output ranges be centered around 0 for the signs to work out properly, or centered about a non-negative number with all non-negative range?  It seems it must be centered about zero, but figured i should check.

thank you !

**Comment by ShubhamBhatt** [comment]
Yes, the center point is 0 right sides are +ve number left side is -ve. Like Graph paper.
